### DISCUSSION TITLE
Automation on 1.1.1 and Live 11

### DISCUSSION THREAD

**Discussion by dhstsw** [apohl79/audiogridder#485] [General]
Hi,

Sorry if i ask again, but did anyone manage to have automation working with client Intrument running on Live 11?

If yes, how? On my system (automation assigned to the target parameter), Live doesn't 'sense' anything when in "configure" on the AudioGridder plugin instance, both using plugin gui or AG semplified one.

Any hint?

**Comment by dhstsw** [comment]
So, i think i found out why it's not working.
Most daw allow you to select the "assigned modulation destination" (may it be pre-assigned or assignable like in AudioGridder [or, for example, Falcon or even Vienna]).
Live works differently: it expect you to click (or move) on the (pre-assigned or assigned) parameter on the GUI of the plugin to detect the parameter you wan to automate after clicking on the "configure" option:
![image](https://user-images.githubusercontent.com/59790194/114276298-b32a5080-9a2e-11eb-8ba6-75f1fd111f4a.png)

This only works if the automation communication between the host and the plugin is bidirectional.

It seems that AudioGridder is only monodirectional (from host to plugin), so, as things are, it will never work with Live.

Anyone can confirm?

thanks.

**Reply by apohl79** [reply]
please check if 1.2 beta3 resolves this for you #537

**Reply by dhstsw** [reply]
Perfectly! :)

Thanks.

Now i spotted another issue (wich i guess only arise with Live) but i have to investigate it further before reporting it.

Thanks!
